components/BiospecimenTable.tsx

```diff
@@ -9,21 +12,6 @@ interface IBiospecimenTableProps {
     synapseAtlases: Atlas[];
 }
 
-const sortFunction = (rows: any[], field: string, direction: any) => {
```

Comment:
This was only working for `HTANBiospecimenID` and only in ascending order. When adding a custom `sortFunction` we need to make sure that we sort the rows for all fields and for all directions.